modules/pcicclient/src/libifm3d_pcicclient/pcicclient_impl.hpp

```diff
@@ -556,35 +562,33 @@ ifm3d::PCICClient::Impl::Call(const std::string& request,
     call_thread_->join();
 
   // Check the return value of our PCIC Call
+  auto predicate = [&] { return has_result.load(); };
```

Comment:
May I ask why there is a catch all by reference instead a explicit capture of ``has_result`` in the lambda?

Reply:
you're correct; will change